modules/s3-api-compatible-services-configuring-acs.adoc

```diff
@@ -21,7 +21,7 @@ To create a new bucket with required permissions, see your S3 provider documenta
 . For *Schedule*, select the backup frequency as daily or weekly, and select the time to run the backup process.
 . Enter the *Bucket* name where you want to store the backup.
 . Optionally, enter an *Object Prefix* if you want to save the backups in a specific folder structure.
-. Enter the *Endpoint* under which the S3 compatible service is reached. It must be reachable via https.
+. Enter the *Endpoint* under which the S3 compatible service is reachable. Defaults to https if the scheme is left out.
```

Comment:
I'm kind of unsure of what this means - does it mean that RHACS attempts to use https for the connection to the endpoint if no connection type is specified? (I'm not sure what "scheme" means in this context.)

Reply:
Yes, in `https://play.min.io` the scheme is `https`. It means that if you enter `play.min.io`, the backend will change this to `https://play.min.io`.